stencil-library/src/components/bank-account-form/example.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
This and `card-form/example.html` new files but the contents are copied from the previous all-in-one example. In this PR a change was made to serve examples as separate files.